experimental/dds/tree2/src/feature-libraries/defaultFieldKinds.ts

```diff
@@ -616,6 +486,40 @@ const optionalFieldEditor: OptionalFieldEditor = {
 	},
 };
 
+const valueFieldEditor: ValueFieldEditor = {
+	...optionalFieldEditor,
+	set: (newContent: ITreeCursor, id: ChangesetLocalId): OptionalChangeset =>
+		optionalFieldEditor.set(newContent, false, id),
+};
+
+const valueChangeHandler: FieldChangeHandler<OptionalChangeset, ValueFieldEditor> = {
+	rebaser: optionalChangeRebaser,
+	codecsFactory: makeOptionalFieldCodecFamily,
+	editor: valueFieldEditor,
+
+	intoDelta: (change: OptionalChangeset, deltaFromChild: ToDelta) =>
+		optionalFieldIntoDelta(change, deltaFromChild),
+	isEmpty: (change: OptionalChangeset) =>
+		change.childChange === undefined && change.fieldChange === undefined,
+};
```

Comment:
Would be nice not to have to repeat stuff that's already on the optional field `FieldChangeHandler`. Sorry I didn't think of that earlier.

```suggestion
const valueChangeHandler: FieldChangeHandler<OptionalChangeset, ValueFieldEditor> = {
	...optional.changeHandler,
	editor: valueFieldEditor,
};
```

Reply:
I think I tried to do this and got into a situation where optional needs value to exist and value needs optional to exist for this to work (essentially circular dependency) because optional's `brandedFieldKind` call uses value's identifier as one of its arguments.

Reply:
You can either replace `new Set([value.identifier])` with `new Set([])` (because we don't use that stuff yet), or extract the "Value" string passed to `brandedFieldKind` into a const.